delete_non_appearing_companions.sql

```diff
@@ -0,0 +1,14 @@
+WITH NonAppearingCompanions(id)
+AS (
+	SELECT c.CompanionID
+	FROM 
+		Companion AS c left join
```

Comment:
fix the format
```suggestion
		Companion AS c
		 LEFT JOIN
```
starting the join in a new line would make the query more readabl, and make sure to use All-Capital for the SQL reserve words